app/controllers/account_automatic_topup_configs_controller.rb

```diff
@@ -1,27 +1,34 @@
 # frozen_string_literal: true
 
 class AccountAutomaticTopupConfigsController < ApplicationController
-  def new; end
-
-  def edit
-    @account = Account.find(params[:account_id])
-    @automatic_topup_config = @account.automatic_topup_configs.find(params[:id])
+  def new
+    @automatic_topup_config = AccountAutomaticTopupConfig.new
```

Comment:
to follow the pattern please create `new_automatic_topup_config` helper method and use it in views.